.github/workflows/darker.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-python@v4
         with:
-          python-version: '3.12'
+          python-version: "3.12"
       - uses: akaihola/darker@1.7.2
         with:
-          version: "1.7.2"
+          version: "2.1.1" # This will use the 2.1.1 darker Python package while using the 1.7.2 darker github action
```

Comment:
I'm surprised that this works, I would have assumed the version has to be < the tag in the "uses" entry. In any case why do we not change both spots?

```suggestion
      - uses: akaihola/darker@2.1.1
        with:
          version: "2.1.1"
```

Reply:
Digging more into this. The github actions is reading from github branches. And the naming convention was updated to include `v` at the start of the version number.
I update now.